### PR TITLE
Task details will only show up if task is selected

### LINKED ISSUE #57: Modify UI such that event details are shown only if it is selected
More tasks can be shown without scrolling/filtering

### PR DESCRIPTION
**Pull request by Skaty**
Resolves #57.

### CODE REVIEW

File: src/test/java/guitests/guihandles/TaskCardHandle.java [hunk 1]
```diff
@@ -24,25 +23,31 @@ public TaskCardHandle(GuiRobot guiRobot, Stage primaryStage, Node node){
...
+    public boolean isDetailsShown() {
+        Node openTimeNode = getNode(OPEN_TIME_FIELD_ID, node);
+        Node closeTimeNode = getNode(CLOSE_TIME_FIELD_ID, node);
+        return openTimeNode.isVisible() && openTimeNode.isManaged() && closeTimeNode.isVisible() && closeTimeNode.isManaged();
+    }
 
     public boolean isSameTask(ReadOnlyTask task){
-        return getFullName().equals(task.getName().taskName);
+        return getTaskName().equals(task.getName().taskName);
 //                && getPhone().equals(person.getPhone().value)
```
Comment (advaypal) [inline src/test/java/guitests/guihandles/TaskCardHandle.java]: Could you delete the comments too please? Thanks

File: src/test/java/guitests/guihandles/TaskCardHandle.java [hunk 2]
```diff
@@ -51,14 +56,14 @@ public boolean isSameTask(ReadOnlyTask task){
     public boolean equals(Object obj) {
         if(obj instanceof TaskCardHandle) {
             TaskCardHandle handle = (TaskCardHandle) obj;
-            return getFullName().equals(handle.getFullName());
+            return getTaskName().equals(handle.getTaskName());
```
Comment (advaypal) [inline src/test/java/guitests/guihandles/TaskCardHandle.java]: Could you delete the comments here too please? Thanks

**Comment by SukiTsang** [comment]
I think it's good now. But still wait until Advay also approves

**Review by SukiTsang — APPROVED** [review]

**Comment by advaypal** [comment]
@Skaty Looks good to me. Also, could you give me merge access? I would have merged your PR in

**Pull request merged**